doc/api/packages.md

```diff
@@ -834,12 +834,12 @@ The following fields in `package.json` files are used in Node.js:
 
 * [`"name"`][] - Relevant when using named imports within a package. Also used
   by package managers as the name of the package.
+* [`"main"`][] - The default module when loading the package, if exports is not
+  specified, and in versions of Node.js prior to the introduction of exports.
 * [`"type"`][] - The package type determining whether to load `.js` files as
   CommonJS or ES modules.
 * [`"exports"`][] - Package exports and conditional exports. When present,
   limits which submodules can be loaded from within the package.
-* [`"main"`][] - The default module when loading the package, if exports is not
-  specified, and in versions of Node.js prior to the introduction of exports.
 * [`"imports"`][] - Package imports, for use by modules within the package
```

Comment:
Maybe note which fields are experimental in their entries in this list so it's a bit more obvious why things are ordered the way they are?

Reply:
Since the stabilization commit is likely a fast follow do you think that is necessary?

Reply:
Nah, not necessary.